src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -101,13 +101,13 @@ dbg_trap="$(trap -p DEBUG | cut -d' ' -f3 | tr -d \')"
 if [[ -z "$dbg_trap" ]]; then
 	__vsc_preexec_only() {
 		__vsc_status="$?"
-		__vsc_preexec
+		eval ${__vsc_preexec}
```

Comment:
this is what fixes it, so I believe we also want to be calling the functions below similarly. @Tyriar can you test with starship?

Reply:
It looks like we don't need the eval before `$dbg_trap`

Reply:
hm, I can't get any debug trap to work now